torch/csrc/jit/codegen/cuda/ir_internal_nodes.h

```diff
@@ -662,10 +716,54 @@ class TORCH_CUDA_CU_API IterDomain : public Val {
     return definition() == nullptr;
   }
 
+  bool isInstruction() const {
+    return is_instruction_;
+  }
+
+  bool isWarpMapped() const {
+    return is_warp_mapped_;
+  }
+
  protected:
   friend TensorDomain;
   friend ReplayTransformations;
   friend IndexReferenceReplay;
+  friend mma_util::WarpMmaSwizzler;
+
+ private:
+  //! Used by WarpMmaSwizzler, marks that this id represents a
```

Comment:
It sounds like a parallel type. Would it make sense to have a new parallel type such as `ParallelType::Mma`?

Reply:
This could be a parallel type since `vectorize` is. But there're also 2 things that makes this a quite different parallel type:

1. User/scheduler cannot just "parallelize" any axis to be "ParallelType::Mma". It just wouldn't be as meaningful without enough context. 

2. Each tensor would have multiple axes that are "ParallelType::Mma", which I don't know if could happen in any other parallel types.

Reply:
I don't understand why #1 is a problem.

For #2, that shouldn't be the case with the current parallel types, but I don't see any problem with it either.